declarations.d.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars */
```

Comment:
I noticed there's `eslint:recommended` in the ESLint config. It has some rules that have false positives in TypeScript — they're fixed in [`@typescript-eslint/recommended`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/eslint-recommended.ts).